### ISSUE TITLE
Installation behind corporate proxy

### ISSUE THREAD

**Issue by troymccormick** [katzgrau/getsparks.org#5]
**TL;DR** It can't be done currently.

Ok...so I'm attempting to get sparks installed on my machine here at work and am unable to do so as I sit behind a corporate proxy server. I get the curl command to run when I input my proxy info properly per the curl documentation, but then I get a fail when it attempt to "Pull down spark manager from http://getsparks.org/static/install/spark-manager-0.0.5.zip". The warning is:

"Warning: copy(http://getsparks.org/static/install/spark-manager00.0.5.zip): failed to open stream: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. in Command line code on line 11"

I'm certain this is because I'm behind said proxy server and the PHP script is attempting to copy down a file, but isn't aware of my proxy settings.

So...in the mean time I've installed it manually, and may have to do that for each spark I want to install, which kind of stinks as I'm starting to love the command line more and more as I use it!  :)

Thanks for reading!

**Comment by seejohnrun** [comment]
Hey - I noticed that the copy() you're using there seems to have the wrong url.  Any chance that's the issue?
We can definitely add proxy support to the loader otherwise

**Comment by troymccormick** [comment]
I assumed that the copy command was being run on it's on from whatever it is you had me curl download in the installer, but I could be mistaken there...I'm not sure...is that something you can check?

**Comment by seejohnrun** [comment]
It does, and the URL is correct in the loader (just checked).  Odd that yours is different.. - that's an exact copy of the output, right?

**Comment by troymccormick** [comment]
Bah...bad bug report on my part. Here's the full content of the output:

```
$ "c:/wamp/bin/php/php5.3.5/php.exe" -r "$(curl -x http://{username}:{password}@{proxyaddress}:80 http://getsparks.org/go-sparks)"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1163  100  1163    0     0    652      0  0:00:01  0:00:01 --:--:--   658
Pulling down spark manager from http://getsparks.org/static/install/spark-manager-0.0.5.zip ...

Warning: copy(http://getsparks.org/static/install/spark-manager-0.0.5.zip): failed to open stream: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 in Command line code on line 11
```

In case it matters, though I doubt it does, I'm using mintty as my console...on Windows obviously... :-)
